docs/ossm/ossm2-migration/cert-manager/README.md

```diff
@@ -0,0 +1,310 @@
+# OpenShift Service Mesh 2.6 --> 3.0 Migration with Cert-Manager
+
+When migrating from OpenShift Service Mesh 2.6 --> 3.0 while using Cert-Manager you can largely follow the [ClusterWide](TODO) or [MultiTenant](../multi-tenancy/README.md) migration guides with a few extra steps detailed here.
+
+<!--
+
+Steps for testing:
+
+1. Install cert-manager operator
+
+1. Install cluster issuer:
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: selfsigned-root-issuer
+  namespace: cert-manager
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: root-ca
+  namespace: cert-manager
+spec:
+  isCA: true
+  duration: 21600h # 900d
+  secretName: root-ca
+  commonName: root-ca.my-company.net
+  subject:
+    organizations:
+    - my-company.net
+  issuerRef:
+    name: selfsigned-root-issuer
+    kind: Issuer
+    group: cert-manager.io
+---
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: root-ca
+spec:
+  ca:
+    secretName: root-ca
+```
+
+1. Install istio-ca
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: istio-ca
+  namespace: istio-system
+spec:
+  isCA: true
+  duration: 21600h
+  secretName: istio-ca
+  commonName: istio-ca.my-company.net
+  subject:
+    organizations:
+    - my-company.net
+  issuerRef:
+    name: root-ca
+    kind: ClusterIssuer
+    group: cert-manager.io
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: istio-ca
+  namespace: istio-system
+spec:
+  ca:
+    secretName: istio-ca
+```
+
+1. Helm install istio-csr
+
+```console
+helm repo add jetstack https://charts.jetstack.io --force-update
+helm upgrade cert-manager-istio-csr jetstack/cert-manager-istio-csr \
```

Comment:
Sorry for complication but since this is 2.6 -> 3.0 migration guide, why not just following the 2.6 OSSM [documentation](https://docs.openshift.com/container-platform/4.16/service_mesh/v2x/ossm-security.html#ossm-cert-manager-installation_ossm-security) as a starting point? That way we will have the same starting point as our customers.

Reply:
That makes sense. I was following the [3.0 OSSM docs for cert-manager](https://docs.redhat.com/en/documentation/red_hat_openshift_service_mesh/3.0.0tp1/html/installing/ossm-cert-manager-assembly#ossm-installing-cert-manager_ossm-cert-manager-assembly).